examples/history-api-fallback/README.md

```diff
@@ -21,13 +21,41 @@ Usage via CLI:
 npx webpack serve --open --history-api-fallback
 ```
 
-_Note: some URLs don't work by default. For example; if the url contains a dot.
+## What Should Happen
+
+1. The script should open `http://0.0.0.0:8080/` in your default browser.
+2. You should see text on the page that reads `Current Path: /`.
+3. Navigate to `http://localhost:8080/foo-bar`.
+4. You should see text on the page that reads `Current Path: /foo-bar`.
+
+_Note: some URLs don't work by default. For example, if the url contains a dot (`/file.txt` in this example).
 Be sure to checkout the [connect-history-api-fallback](https://github.com/bripkens/connect-history-api-fallback)
 options._
 
+To allow `/file.txt` to fallback, update the `webpack.config.js` as follows:
+
+**webpack.config.js**
+
+```js
+module.exports = {
+  // ...
+  devServer: {
+    historyApiFallback: {
+      disableDotRule: true,
+    },
+  },
+};
+```
+
+Usage via CLI:
```

Comment:
Doesn't `Usage via CLI` section usually describe how to acheive the same configuration with command line arguments?

Reply:
Yes, but here we don't have cli arguments to pass object options to `histpryApiFallback`.

Maybe we can reword it as - 

```suggestion
then to run the example use the following command:
```